webapi/src/main/scala/org/knora/webapi/util/ConstructResponseUtilV2.scala

```diff
@@ -841,7 +840,7 @@ object ConstructResponseUtilV2 {
             case (acc: Set[IRI], (_: SmartIri, valObjs: Seq[ValueRdfData])) =>
                 val mappings: Seq[String] = valObjs.filter {
                     valObj: ValueRdfData =>
-                        valObj.valueObjectClass == OntologyConstants.KnoraBase.TextValue.toSmartIri && valObj.assertions.get(OntologyConstants.KnoraBase.ValueHasMapping.toSmartIri).nonEmpty
+                        valObj.valueObjectClass == OntologyConstants.KnoraBase.TextValue.toSmartIri && valObj.assertions.contains(OntologyConstants.KnoraBase.ValueHasMapping.toSmartIri)
```

Comment:
same here ;-)

Reply:
Eventually IntelliJ will do the programming for us. :)